src/vaadin-text-field-mixin.html

```diff
@@ -297,7 +297,11 @@
         },
 
         /**
-         * When true, the input element has a non-empty value entered by the user.
+         * Whether the input element has user input.
+         *
+         * Note, the property indicates true only if the input has been entered by the user.
+         * In the case of programmatic changes, the property must be reset to false.
+         *
          * @protected
          */
```

Comment:
I later changed this JSDoc again 🤦 . Let's use the last revision:

https://github.com/vaadin/web-components/pull/5645/files#diff-ad9af27069d2b2d9c396891b6568a8aa654857ee690a61c29479274e04944e60L56-L63